Fitz/Features/AccountsRework/Create/Domain/CreateAccountService.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace Fitz.Features.AccountsRework.Create.Domain
+{
+    public class CreateAccountService
+    {
+        public CreateAccountModel BuildAccount(CreateAccountCommand command)
+        {
+            var CreateAccountCommand = this.ValidateCreateAccountCommand(command);
+
+            return new CreateAccountModel
+            {
+                Id = CreateAccountCommand.Id,
+                Username = CreateAccountCommand.Username,
+                Beer = 0,
+                LifetimeBeer = 0,
+                SafeBalance = 0,
+                Favorability = 100,
+                CreatedDate = CreateAccountCommand.CreatedDate,
+                LastSeenDate = DateTime.Now,
+                LastActivityDate = DateTime.Now,
+                SubscribeToLottery = false,
+                SubscribeTickets = 0,
+                Deactivated = false,
+            };
+        }
+
+        private CreateAccountCommand ValidateCreateAccountCommand(CreateAccountCommand command)
+        {
+            return command;
```

Comment:
I didn't really know what needs validated for this command yet but wanted to demonstrate that this pattern would persist with the rest of the feature refactors.